### PR TITLE
Player Spider Fixes

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Maybe a feature? It's a fix, IMO.

Makes it so that player spiders who are spawned in via sentience potions get a notification very similar to the sentience potion; it notifies them that the original user of the sentience potion (on the original spider) is their master and that they should obey+protect them.

If badmins make player spiders, this notification does *NOT* pop up, meaning you can go full ham, technically (if in doubt, ahelp!)

Basically--if you see this notification as a spider, don't go around attacking everyone on sight unless your master or your fellow spiders (who got orders from the master) tells them to.

Code Wise

- Added a new master_commander var, which can be used for things like sentience options; this has a side effects of making it much easier to track who created sentient mobs, as well.

### CODE REVIEW

**Comment by ZomgPonies** [comment]
Should compile, seems travis -> github connection shat during test.
